jujugui/static/gui/src/app/app.js

```diff
@@ -1139,12 +1139,21 @@ YUI.add('juju-gui', function(Y) {
       // Auto place the units. This is probably not be best UX, but is required
       // to display the machines in the deployment flow.
       autoPlaceUnits();
+      const credentialTag = env.get('credentialTag');
+      let cloud = env.get('cloud');
+      if (cloud) {
+        cloud = {id: cloud, name: cloud};
+      }
       ReactDOM.render(
         <window.juju.components.DeploymentFlow
           acl={this.acl}
           changesFilterByParent={
             changesUtils.filterByParent.bind(changesUtils, currentChangeSet)}
           changeState={this.changeState.bind(this)}
+          cloud={cloud}
+          credential={
+            credentialTag ? credentialTag.replace('cloudcred-', '') : undefined}
```

Comment:
If you could add a comment here to the effect of "Removing juju tag from credential name....." 

Just so future us knows what's going on here.